### PR TITLE
Feat/home pg button

### LINKED ISSUE #1: Major UI update
Add a sleek and clean UI for the whole thing

### LINKED ISSUE #2: Add return button
Add a return to homepage button or clickable homepage header within the page that the user can click on to move back to the homepage.

### PR DESCRIPTION
**Pull request by PrajaktaUpadhye6**
#3 Resolved
Completed suggested changes related to directory.
Shifted index.html page to root directory.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 <body>
     <h1>Odin Recipes</h1>
-    <img src="./img/mad-cook.jpg" alt="mad cook experimenting" width="60%">
+    <img src="../img/mad-cook.jpg" alt="mad cook experimenting" width="60%">
```
Comment (codedstrings) [inline index.html]: please make the src = "./img/mad-cook.jpg"

**Review by codedstrings — CHANGES_REQUESTED** [review]
The image path was updated from "./img/mad-cook.jpg" to "../img/mad-cook.jpg." Since index.html is in the root directory, the original path ./img/mad-cook.jpg is correct. The ../ would look for the img folder outside of the current directory, which isn't accurate in this case. Could you please change it back?

Also, I noticed you closed the previous PR and created a new one. In the future, it's usually easier to continue committing changes to the same PR instead of creating a new one. This helps keep the discussion and review process organized in one place. You just have to commit in the PR branch (feat/HomePg-button) as you normally would, and the changes would reflect in this PR. 

Everything else looks fine to me! Please add the requested change so I can Merge the PR. (be sure to not create a new PR)

**Comment by codedstrings** [comment]
@PrajaktaUpadhye6 Hey, I would like to close the issue soon...It'll be greatly helpful if you could please add the requested changes soon.

**Comment by PrajaktaUpadhye6** [comment]
Can you check the suggested changes, I have also tried to enhance the UI a bit

**Review by codedstrings — COMMENTED** [review]
LGTM! Thanks so much for the changes—I love the UI improvements, they look great! 🎉 I’ve linked this PR to a new issue for the UI changes, though it’s usually better to keep different tasks in separate PRs for easier tracking, No issue here tho!

One small suggestion: instead of adding styles directly to each HTML file, it’s better to use a single styles.css for easier maintenance. Feel free to update this if you'd like! Also, you're welcome to add more UI changes anytime—your contributions are much appreciated!

**Review by codedstrings — APPROVED** [review]

**Pull request merged**